### PR TITLE
Fix(T28449): add missing prop to DpMultiselect

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T28449

**Description:** This PR adds the missing prop `allowEmpty` to `dp-multiselect`, which prevents the deselection of values.
[
PR152 in demospipes](https://github.com/demos-europe/demosplan-addon-demospipes/pull/152)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Added
 
+- ([#721](https://github.com/demos-europe/demosplan-ui/pull/721)) Pass the allowEmpty prop to vue-multiselect, which prevents the deselection of values ([@sakutademos](https://github.com/sakutademos))
```
Comment (muellerdemos) [inline CHANGELOG.md:9]: this should be under:
### Fixes
since this is a bug fix and not adding a feature

**Review by muellerdemos — APPROVED** [review]
code lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (spiess-demos) [reply CHANGELOG.md:9]: from a demosplan-ui perspective, it is a new feature of the DpMultiselect component.

Comment (muellerdemos) [reply CHANGELOG.md:9]: makes sense, I agree

**Review by elmasdemos — APPROVED** [review]
lgtm

**Pull request merged**